test/api/v3/unit/libs/payments.test.js

```diff
@@ -55,6 +61,21 @@ describe('payments/index', () => {
       expect(difference - 60).to.be.lessThan(3); // the difference is approximately two months, +/- 2 days
     });
 
+    it('data.nextBill is undefined', () => {
```

Comment:
If you can point me at a good example file or let me know what the standard is, I can update this file to match it.